crates/iota-core/src/unit_tests/authority_tests.rs

```diff
@@ -2069,15 +2069,15 @@ async fn test_conflicting_transactions() {
             &ok.clone().status.into_signed_for_testing(),
             object_info
                 .lock_for_debugging
-                .expect("object should be locked")
+                .expect("object is not locked")
                 .auth_sig()
         );
 
         assert_eq!(
             &ok.clone().status.into_signed_for_testing(),
             gas_info
                 .lock_for_debugging
-                .expect("gas should be locked")
+                .expect("gas is not locked")
```

Comment:
where did that changed? :D

Reply:
So basically this is suggested by @DaughterOfMars [here](https://github.com/iotaledger/iota/pull/4757#discussion_r1913411569) where
```bash
1. These messages should be all lowercase
2. The phrasing should be "locks are not set" instead of "locks should be set", I.E. Descriptive not prescriptive
```